client/sites-dashboard/components/sites-content-controls.tsx

```diff
@@ -135,12 +135,24 @@ export const SitesContentControls = ( {
 					selectedText={ sprintf( __( 'Status: %(siteStatus)s' ), {
 						siteStatus: selectedStatus.title,
 					} ) }
+					ariaLabel={ sprintf(
+						// Translators: `siteStatus` is one of the site statuses specified in the Sites page.
+						__( 'Filtering by Status "%(siteStatus)s". Switch status filter.' ),
```

Comment:
I'm not sure `Switch status filter` is necessary in this case. I think I'd prefer:
```
__( 'Filtering to sites with status "%(siteStatus)s"' )
```

Additionally, when we're looking at all sites, we should use a different label:

```
__( 'Displaying all sites.' )
```

Reply:
Good point, I changed those. "Switch status filter" CTA seems redundant indeed as the screen reader will read, "You are currently on the button [...]" text after it reads the label.